server/setup.go

```diff
@@ -37,28 +37,6 @@ var (
 	)
 )
 
-// Setup connects to a database and starts serving requests
-func Setup() error {
-	loadVarsFromINI()
-	log.Printf("Using %s for persistence\n", *database)
-
-	ctx := context.Background()
-	dbFactory, err := getDBFactory(ctx, factoryConfig{
-		canRunCreateStmts: true,
-	})
-	if err != nil {
-		return err
-	}
-	cs := newCribbageServer(dbFactory)
-	err = seedNPCs(ctx, dbFactory)
-	if err != nil {
-		return err
-	}
-	cs.Serve()
-
-	return nil
-}
```

Comment:
This feels like the wrong place to split based on build flags. Why don't we split behavior in the server package?

Reply:
what if we did something like this instead? https://github.com/joshprzybyszewski/cribbage/compare/splitServerlessElsewhere?expand=1

Reply:
You _can_ do that, like I've done [here](https://github.com/cszczepaniak/oh-hell-backend/blob/main/app/main.go). However, the env var felt slightly messier:
- What if AWS changes the name of the var that I use in that link?
- If we don't use AWS's var, what if we forget to set our own? Is that just as easy as forgetting to build with the right tag?
- We never need to change this dynamically
- We save ourselves a branch/some binary size by compiling the setting in (ok, that's not that big of a deal, but still)

But I see your point; this is probably too large of a split. How would you feel about _minimizing_ the two files that have mutually exclusive build tags (rough sketch below)?

`lambda.go`:
```go
// +build lambda

package server

const inLambda = true

func serve(h http.Handler) error {
    return gateway.ListenAndServe(`:8080`, h)
}
```
`nonlambda.go` (or some better file name):
```go
// +build !lambda

package server

const inLambda = false

func serve(h http.Handler) error {
    return http.ListenAndServe(`:8080`, h)
}
```

Reply:
jk, I did the env var thing :shrug:

Reply:
lol oops all i have to do is leave it alone for 11 days:#